couchbase.py

```diff
@@ -303,6 +304,20 @@ def _parse_metrics(obj_to_parse, dimensions, request_type, module_config):
                     if metric:
                         metrics.append(metric)
 
+        if 'hot_keys' in obj_to_parse:
+            hot_keys = obj_to_parse['hot_keys']
+            key = 0
+            metric_name_pref = 'bucket.hot_keys'
+            for keys in hot_keys:
+                for name, value in keys.items():
+                    if isinstance(value, numbers.Number):
+                        key += 1
+                        metric = _process_metric(metric_name_pref, str(key),
```

Comment:
I would add the actual name of the key to the `dimensions` dict.  The dict will have to be (shallow) copied first so that it doesn't get added to other metrics unrelated to hot keys.  This way, we can know what key a certain rank corresponds to.